schema-guide.md

```diff
@@ -193,104 +205,174 @@ commit: "Revision: 8612"
 
 ### `contact`
 
-- type: ...
-- required: false
+- type: Array of [`definitions.person`](#definitionsperson) and/or [`definitions.entity`](#definitionsentity) objects.
+- required: `false`
 
-...
+The contact person, group, company, etc. for the software or dataset.
 
-Usage example:
+Usage examples:
 
 ```yaml
+contact:
+  - affiliation: "Humboldt-Universität zu Berlin"
+    email: "mail@sdruskat.net"
+    family-names: Druskat
+    given-names: Stephan
+```
+```yaml
+contact:
+  - email: "mail@research-project.org"
+    name: "The Research Software project"
+```
+```yaml
+contact:
+  - email: "mail@sdruskat.net"
+    given-names: Stephan
+    family-names: Druskat
+  - email: "mail@research-project.org"
+    name: "The Research Software project"
 ```
 
 ### `date-released`
 
-- type: ...
-- required: false
+- type: [`definitions.date`](#definitionsdate)
+- required: `false`
 
-...
+The date the software or data set has been released. Format is 4-digit year, 2-digit month, 2-digit day of month, separated by dashes.
 
 Usage example:
 
 ```yaml
+date-released: 2020-01-31
 ```
 
 ### `doi`
 
-- type: ...
-- required: false
+- type: [`definitions.doi`](#definitionsdoi)
+- required: `false`
 
-...
+The DOI of the software or data set. This notation is most useful when there is just one DOI you want to include. In
+that case, `doi` can be used as shorthand for something like:
 
-Usage example:
+```yaml
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.1003149
+    description: The concept DOI of the work.
+```
+or
 
 ```yaml
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.4813122
+    description: The versioned DOI of the work.
 ```
 
-### `identifiers`
+Usage examples:
 
-- type: ...
-- required: false
+```yaml
+doi: 10.5281/zenodo.1003149
+```
+```yaml
+doi: 10.5281/zenodo.4813122
+```
 
-...
+### `identifiers`
 
-Usage example:
+- type: array of [`definitions.identifier`](#definitionsidentifier) objects.
+- required: `false`
 
-```yaml
-```
+The identifiers of the software or dataset.
+
+Usage example: see [`definitions.identifier`](#definitionsidentifier).
 
 ### `keywords`
 
-- type: ...
-- required: false
+- type: array of `string`
+- required: `false`
 
-...
+Keywords that describe the work.
 
 Usage example:
 
 ```yaml
+keywords:
+ - thefirstkeyword
+ - thesecondkeyword
+ - and a third
```

Comment:
```suggestion
 - a third keyword
```

Just to make this complete and in line with the two examples above, taken this is meant to show that you can use a string with spaces here.